airflow/jobs.py

```diff
@@ -2234,13 +2235,23 @@ def _process_backfill_task_instances(self,
                                 # Skip scheduled state, we are executing immediately
                                 ti.state = State.QUEUED
                                 session.merge(ti)
+
+                                # no need to clean temp file, airflow run cli removes
```

Comment:
It's a bit unclear what this is referring to, and the line of code sited could change. I think it's fine to just remove this comment